public/app/features/alerting/unified/hooks/useMuteTimingOptions.ts

```diff
@@ -13,8 +13,10 @@ export function useMuteTimingOptions(): Array<SelectableValue<string>> {
   const config = currentData?.alertmanager_config;
 
   return useMemo(() => {
+    // merge both fields mute_time_intervals and time_intervals to support both old and new config
+    const time_intervals = [...(config?.mute_time_intervals ?? []), ...(config?.time_intervals ?? [])];
```

Comment:
Should we create a function which merges time interval from the config? The same merging code is repeated in different places